Bottom: b691f3ed96b4510c220f5a4ecf3d8526ab4a6495
Top:    366c7831dd7fb4941504754c2759441128ea1e1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:58:51 +0100

Refresh of action-c-remove-unused-action-

---

diff --git a/src/action.h b/src/action.h
index ee116e8..e92c578 100644
--- a/src/action.h
+++ b/src/action.h
@@ -35,7 +35,6 @@
 
 #define CLONE_TYPES LINE_TYPE | ARC_TYPE | VIA_TYPE | POLYGON_TYPE
 
-void ActionMovePointer (char *, char *);
 void ActionAdjustStyle (char *);
 void EventMoveCrosshair (int, int);
